python/lsst/sims/utils/fileMaps.py

```diff
@@ -0,0 +1,81 @@
+import re
+
```

Comment:
The Sed filenames on fatboy do not include the sub-directory of SIMS_SED_LIBRARY_DIR (i.e. fatboy contains 'kp50.90.fits.gz' when the actual file is in 'starSED/kurucz/kp50.90.fits.gz').  This is just a dict that maps the one to the other.

There probably isn't a logical place for this class to be.  I can imagine someone wanting to use this class for their own non-fatboy database, so I will leave it here for the time-being.